app/components/preprint-form-authors.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
```

Comment:
At minimum, Dawn's pattern of putting styles in a separate file (not app.scss) is highly valuable and recommended. The rest... well, I did include downsides for a reason. Neither option is amazing. :)